src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -145,6 +145,10 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 			}
 			this._editableDropdown.enabled = this.enabled;
 			this._editableDropdown.fireOnTextChange = this.fireOnTextChange;
+
+			if (this.showInputTooltip) {
+				this._editableDropdown.input.setTooltip(this._editableDropdown.value);
```

Comment:
Did you test : 

1. That the tooltip is updated appropriately when a new value is selected from the dropdown
2. That the tooltip is updated appropriately as the user types in text

Reply:
This only adds tooltip to the editable dropdown. I think we should add it to the non-editable version as well.

Reply:
As the PR description mentions - the non-editable dropdown already has the tooltip set by default. So there's no changes needed there.

Reply:
@Charles-Gagnon Based on the your feedback, I realized a tooltip is only necessary when the editable dropdown is disabled. When disabled, the user won't be able to move the cursor within the box to see the overflow text.

I've kept the `showInputTooltip` property so we can choose if a disabled dropdown shows a tooltip on hover. I can also completely remove the `showInputTooltip` property so that disabled editable dropdowns always have the tooltip. What are your thoughts?